apps/apps.c

```diff
@@ -3198,6 +3198,32 @@ int app_isdir(const char *name)
 #endif
 
 /* raw_read|write section */
+#if defined(__VMS)
+# include "vms_term_sock.h"
+static int stdin_sock = -1;
+static void close_stdin_sock(void)
```

Comment:
Doesn't style guide suggest empty line prior function? This is a side note, i.e. it's not necessarily to act upon it, you are free to use the plus-one request has already.